src/dom/Value.ts

```diff
@@ -217,10 +217,21 @@ export interface Value {
   ): boolean;
 
   /**
-   * For checking non-strict equivalence of an Ion Values represented by dom.Value with the other value represented
-   * by an JS object.
+   * Compares this instance of dom.Value to the provided value and returns true
+   * if they are equal. If `other` is a dom.Value, this comparison checks for structural
+   * (or "non-strict") equivalence. If `other` is a native JS value, this comparison will
+   * convert it to an Ion value and then check for structural equivalence.
+   *
+   * @param other                       other Ion Value or Js Object to be compared with this Ion Value.
+   * @param options                     options provided for equivalence as below
+   *        epsilon                     used by Float for an equality with given epsilon precision. (Default: null)
```

Comment:
Now that `equals` allows for comparisons across numeric types (e.g. `someFloat.equals(someDecimal)`), should `epsilon` has to be considered across types too?

I'm inclined to say yes for consistency, but `ion.Decimal` doesn't currently make it very easy to do arithmetic so it might take non-trivial effort to implement.